.changeset/rich-ladybugs-own.md

```diff
@@ -0,0 +1,13 @@
+---
+"@shopify/shopify-api": major
+---
+
+> [!NOTE]
+> This change only affects apps that are using custom runtime adapters.
+> If you're using a default adapter from this package, you don't need to make this change.
```

Comment:
~~Worth giving a code example that shows what using a default adapter from this package means? Wouldn't suprise me if partners don't understand what this means, might be better to show them?~~

The migration guide is great, is it worth linking to this here?